docs/reference/run.md

```diff
@@ -900,6 +901,11 @@ If you do block IO in the two containers at the same time, by, for example:
 You'll find that the proportion of time is the same as the proportion of blkio
 weights of the two containers.
 
+The `--blkio-weight-device` flag can set the weight of a particular device to a value between 10 to 1000. With this, override the default value of group weight as specified by blkio.weight.
+For example, the command below create one container and set weight of /dev/sda 
```

Comment:
Yes, it does. It reads a bit ESL. 

---> https://gist.github.com/moxiegirl/ee913ca6ec2af45be074

The `--blkio-weight-device="DEVICE_NAME:WEIGHT"` flag sets a specific device weight.
The `DEVICE_NAME:WEIGHT` is a string containing a colon-separated device name and weight.
For example, to set `/dev/sda` device weight to 200:

    $ docker run -it \
        --blkio-weight-device "/dev/sda:200" \
        ubuntu

If you specify both the  `--blkio-weight` and `--blkio-weight-device`, Docker uses the `--blkio-weight` as the default weight and uses `--blkio-weight-device` to override this default with a new value on a specific device. The following example uses a default weight of `300` and override_udev_sync_checkrides this default on `dev/sda` setting that weight to `200`:

    $ docker run -it \
      --blkio-weight 300 \
      --blkio-weight-device "/dev/sda:200" \
      ubuntu